### ISSUE TITLE
How does this project differ from '256dpi/arduino-mqtt' ?

### ISSUE THREAD

**Issue by njh** [monstrenyatko/ArduinoMqtt#4]
https://github.com/256dpi/arduino-mqtt

**Comment by monstrenyatko** [comment]
@njh 
The main difference is that the [256dpi/arduino-mqtt](https://github.com/256dpi/arduino-mqtt) library is a wrapper around [Eclipse Paho MQTTClient](https://github.com/eclipse/paho.mqtt.embedded-c) when my library is the `C++` reimplementation of the `Paho MQTTClient`.

`Paho MQTTClient` has very bad support and a lot of unresolved issues.
I decided to keep `Paho MQTTPacket` library that covers MQTT packets encoding/decoding but implement own better and low-power friendly `MqttClient` class.
As a bonus, I made the `ArduinoIDE` and `PlatformIO Core` friendly file structure and published to both official markets.
